packaging/pkg_helpers.bash

```diff
@@ -64,6 +64,9 @@ setup_cuda() {
       export FORCE_CUDA=1
       export NVCC_FLAGS="-gencode=arch=compute_35,code=sm_35 -gencode=arch=compute_50,code=sm_50 -gencode=arch=compute_60,code=sm_60 -gencode=arch=compute_70,code=sm_70 -gencode=arch=compute_50,code=compute_50"
       ;;
+    rocm*)
+      export USE_ROCM=1
```

Comment:
Does this need to be `FORCE_CUDA=1` in addition to the USE_ROCM?  IIRC, at least for torchvision, it tries to detect CUDA/ROCm at build-time and fails since the build is done on a CPU-only machine, so you have to tell the job to build for CUDA even though it wasn't detected.

Reply:
I will make the change